### ISSUE TITLE
Black screen with lr-pcsx-rearmed but not with pcsx-rearmed

### ISSUE THREAD

**Issue by GiulioCentorame** [libretro/pcsx_rearmed#361]
##Description

I am getting confused here. I am able to run anything with pcsx-rearmed, but when I try to use the libretro version I get a black screen.
This happens only with some games and not with others.

##Steps to reproduce

Loaded all the bios (md5 checked as per Wiki page) in the BIOS folder, currently with lowercase names. The folder contains other BIOS files from various emulators. After selecting the game, I get a black screen, and it comes back to the game selection menu.
The only major setting I have changed so far is disable_overscan=1 in the Retropie config file.
Tried with both .cue/bin and .pbp files of the same game (the latter usually works best for me).

##When did the behavior start?

From the first installation

##Your device/OS/platform/architecture

RPi 3B+ 1.3
Retropie 4.4

Edit: fixed typo

**Comment by hizzlekizzle** [comment]
We'll need a log and some examples of games that don't work for you to be able to investigate.

**Comment by negativeExponent** [comment]
.pcp? disable_overscan=1? don't think these are parts of this core. im not sure if git version matches with retropie but you should post it as well, together with whatever hizzlekizzle asked for.

**Comment by GiulioCentorame** [comment]
Apologies, it's *.pbp. The games I have tried so far are:

* Crash Bandicoot (works with no lag)
* Spyro (works lagging)
* Grand Theft Auto
* Castlevania: SOTN
* Metal Gear Solid

I tried the last three of them with both .pbp files and .cue/.bin files, with no apparent difference (same behaviour).

The option I am referring to (disable_overscan) is */boot/config.txt* on RetroPie.

What is the best way of getting logs for it on Raspbian? I will do it as soon as I have access to the device.
